src/Files.OpenDialog/FilesLauncher/FilesLauncher.cpp

```diff
@@ -122,23 +125,22 @@ int WINAPI WinMain(HINSTANCE hInstance,
 
 		// Create the window.
 		HWND hwnd = CreateWindowEx(
-			0,                         // Optional window styles.
-			CLASS_NAME,                // Window class
-			L"Files Launcher",         // Window text
-			0,                         // Window style
-			0, 0, 0, 0,    // Size and position
-			HWND_MESSAGE,  // Parent window (message-only window)
-			NULL,          // Menu
-			hInstance,     // Instance handle
-			&openInFolder  // Additional application data
+			0,
+			CLASS_NAME,
+			L"Files Launcher",
+			0,
+			0, 0, 0, 0,
+			HWND_MESSAGE,
+			NULL,
+			hInstance,
+			&openInFolder
```

Comment:
Is it appropriate to delete the comment?

Reply:
Absolutely, YES. Need C++ dev for review.

Reply:
OK. You're right.